contentscript.js

```diff
@@ -1,37 +1,35 @@
-document.onmouseup = function run(event) {
-  event.stopPropagation();
-
+document.onmouseup = function run(event1) {
   if (window.getSelection() != "") {
+    var keys = [];
+    onkeydown = onkeyup = function(event2) {
+      keys[event2.keyCode] = event2.type == 'keydown';
 
-    xCoord = event.pageX;
-    yCoord = event.pageY;
-    var that = this;
-    spawnedEcho = spawnEchoForm(xCoord, yCoord, that);
+      if ( (keys[17] === true) && (keys[69] === true) ) {
+        var selectedString = window.getSelection().toString();
```

Comment:
Two tweaks to request:
1 - The highlighting currently disappears once the form is popped in with cmd+E (but that pop-in feels good!)
2 - Pop the form in a little bit further down. On the sites I used it, it overlaps the bottom line of text I highlighted. Maybe clear it by another 50px or so?